docs/src/main/asciidoc/spanner.adoc

```diff
@@ -420,6 +420,9 @@ If a property marked for lazy fetching is never retrieved, then it is also skipp
 
 If used inside a transaction, subsequent operations on lazily-fetched properties use the same transaction context as that of the original parent entity.
 
+===== Soft Delete
+The `@Where` annotation could be applied to the entity class or to the interleaved property. This annotation provides an SQL where clause
```

Comment:
Our conventions is one sentence per line.

Reply:
ok, thanks